front/src/pods/hotel-edit/hotel-edit.container.tsx

```diff
@@ -31,7 +46,20 @@ const HotelEditContainerInner = (props: Props) => {
     loadHotel();
   }, []);
 
-  const doSave = () => {};
+  const doSave = () => {
+    HotelEditFormValidation.validateForm(hotel).then(formValidationResult => {
+      if (formValidationResult.succeeded) {
+        //save
+      } else {
+        alert("error, review the fields");
+        const updateHotelFormErrors = {
+          ...hotelFormErrors,
+          ...formValidationResult.fieldErrors
+        };
+        setHotelFormErrors(updateHotelFormErrors);
+      }
+    });
```

Comment:
Hadouken? 

I would rather follow same level of abstraction level, something like (psuedocode):

```typescript
const doSave() {
    HotelEditFormValidation.validateForm(hotel).then(formValidationResult => {
         handleFormValidation(formValidationResult);
    });
}

const handleFormValidation = (formValidation) => {
  if(formValitacion.succeeded) {
     doSaveServerRequest();
  } else {
   showErrorNotification();  
}
}
```